HUD/SetFakePausemapPlayerPositionThisFrame.md

```diff
@@ -0,0 +1,32 @@
+---
+ns: HUD
+aliases: ["0x77E2DD177910E1CF", "_SET_PLAYER_BLIP_POSITION_THIS_FRAME"]
+---
+## SET_FAKE_PAUSEMAP_PLAYER_POSITION_THIS_FRAME
+
+```c
+// 0x77E2DD177910E1CF 0x54E75C7D
+void SET_FAKE_PAUSEMAP_PLAYER_POSITION_THIS_FRAME(float x, float y);
+```
+
+Overrides the position of the main player blip for one frame.
```

Comment:
for the current frame*

Reply:
does such a small difference really matter?